tests/test_operations.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import rdflib
+import requests
+
+from fdpclient import operations
+
+# TODO
+# - add data format tests
+# - add exception tests: invalid input, not found...
+
+# NOTE: change IP address to domain name after solving FDP server config issue
+base_url = 'http://145.100.57.30/catalog'
+data_url = base_url + '/catalog01'
+
+@pytest.fixture(scope='class', autouse=True)
+def setup():
+    r = requests.delete(data_url)
+    yield
+    r = requests.delete(data_url)
+
+class TestDefault:
+    """Test fdpclient.operations functions"""
+
+    # datadir fixture provided via pytest-datadir-ng
+    def test_create(self, datadir):
+        """Test create function"""
+        with open(datadir['catalog01.ttl']) as f:
+            data = f.read()
+        r = operations.create(base_url, data=data)
```

Comment:
This doesn't actually test whether the catalog was created, does it?

Reply:
The "return is None" means it was created, otherwise exceptions would be raised. And it'd be better to also explicitly check the created content :-)

Reply:
Well, unless `operations.create()` fails to raise an exception if something goes wrong, and so makes it appear that it was created successfully but wasn't. I think you want to check that the new object really exists.